components/brave_wallet/common/BUILD.gn

```diff
@@ -203,6 +203,7 @@ rust_static_library("rust_lib") {
     "//brave/components/brave_wallet/browser:hd_keyring",
     "//brave/components/brave_wallet/browser:transaction",
     "//brave/components/brave_wallet/browser:unit_tests",
+    "//brave/components/brave_wallet/browser/polkadota/rust:*",
```

Comment:
Do we actually use it here?

Reply:
This was completely extraneous, even worse because there's a typo in the path as well: `polkadota/rust:*`